app/controllers/posts_controller.rb

```diff
@@ -1,16 +1,12 @@
 class PostsController < ApplicationController
   def index
     # @user = User.includes(:posts).find(params[:user_id])
```

Comment:
- [ ] Kindly remove this commented code. Removing such code makes your work look clean and tidy 💻